dlrs/dlrs.rst

```diff
@@ -1074,65 +973,49 @@ OpenVINO is a trademark of Intel Corporation or its subsidiaries
-
-.. _flannel: https://github.com/coreos/flannel
 
 .. _Getting Started with Kubeflow: https://github.intel.com/verticals/usecases/blob/56717f4642ecd958dc93bbc361c551dfc578d3ed/kubeflow/README.md#getting-started-with-kubeflow
 
-.. _TensorFlow Eigen: https://hub.docker.com/r/sysstacks/dlrs-tensorflow-clearlinux:v0.6.0-oss
-
-.. _TensorFlow oneDNN: https://hub.docker.com/r/sysstacks/dlrs-tensorflow2-clearlinux:v0.6.0
+.. _TensorFlow for Ubuntu: https://hub.docker.com/r/sysstacks/dlrs-tensorflow-ubuntu
 
-.. _PyTorch with OpenBLAS: https://hub.docker.com/r/sysstacks/dlrs-pytorch-clearlinux:v0.6.0-oss
+.. _TensorFlow 2 for Ubuntu: https://hub.docker.com/r/sysstacks/dlrs-tensorflow2-ubuntu
 
-.. _PyTorch with Intel oneDNN: https://hub.docker.com/r/sysstacks/dlrs-pytorch-clearlinux:v0.6.0
+.. _PyTorch for Ubuntu: https://hub.docker.com/r/sysstacks/dlrs-pytorch-ubuntu
 
-.. _Intel oneDNN: https://hub.docker.com/r/sysstacks/dlrs-tensorflow-clearlinux
+.. _DLRS V1.0: https:///deep-learning-reference-stack
 
-.. _DLRS V3.0:  https://clearlinux.org/stacks/deep-learning-reference-stack-v3
+.. _DLRS V2.0: https:///deep-learning-reference-stack
 
-.. _DLRS V4.0: https://clearlinux.org/news-blogs/deep-learning-reference-stack-v4
+.. _DLRS V3.0:  https:///deep-learning-reference-stack-v3
 
-.. _DLRS V5.0: https://clearlinux.org/blogs-news/deep-learning-reference-stack-v50-now-available
+.. _DLRS V4.0: https:///deep-learning-reference-stack-v4
 
-.. _DLRS V6.0: https://clearlinux.org/blogs-news/deep-learning-reference-stack-v6-now-available
+.. _DLRS V5.0: https:///deep-learning-reference-stack-v50-now-available
 
-.. _dlrs-tfjob: github.com/intel/stacks
+.. _DLRS V6.0: https:///deep-learning-reference-stack-v6-now-available
 
-.. _Logging Architecture: https://kubernetes.io/docs/concepts/cluster-administration/logging/
-
-.. _DLRS V1.0: https://clearlinux.org/stacks/deep-learning-reference-stack
-
-.. _DLRS V2.0: https://clearlinux.org/stacks/deep-learning-reference-stack-pytorch
+.. _DLRS V7.0: https:///deep-learning-reference-stack-v7-now-available
 
 .. _Jupyter Notebook: https://jupyter.org/
 
-.. _kubectl overview: https://kubernetes.io/docs/reference/kubectl/overview/
-
 .. _launcher.py: https://github.com/clearlinux/dockerfiles/tree/master/stacks/dlrs/kubeflow
 
-.. _DLRS Terms of Use: https://clearlinux.org/stacks/deep-learning/terms-of-use
+.. _DLRS Terms of Use: https://intel.github.io/stacks/dlrs/terms_of_use.html
 
-.. _DLRS Release notes: https://github.com/intel/stacks/tree/master/dlrs
+.. _DLRS Release notes: https://intel.github.io/stacks/dlrs/clearlinux/releasenote.html
 
 .. _Seldon Core: https://docs.seldon.io/projects/seldon-core/en/latest/
 
-.. _Istio: https://github.com/kubeflow/manifests/blob/master/kfdef/kfctl_k8s_istio.yaml
-
 .. _Dockerfile_openvino_base: https://github.com/clearlinux/dockerfiles/blob/master/stacks/dlrs/kubeflow/dlrs-seldon/docker/Dockerfile_openvino_base
```

Comment:
let's remove this `clearlinux` url as well.

Reply:
done, replaced with pointer to usecase dockerfile.